### PR TITLE
Complete entailment algorithm in Semantics Appendix

### PR DESCRIPTION
**Pull request by Antoine-Zimmermann**
This covers the case of every graph entails `?s rdf:type rdfs:Resource`.

***
<a href="https://pr-preview.s3.amazonaws.com/Antoine-Zimmermann/rdf-semantics/pull/48.html" title="Last updated on May 2, 2024, 2:28 PM UTC (86c2a91)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-semantics/48/65a1f76...Antoine-Zimmermann:86c2a91.html" title="Last updated on May 2, 2024, 2:28 PM UTC (86c2a91)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -1516,6 +1516,7 @@ <h2>Entailment rules (Informative)</h2>
   <ol>
     <li>Add to S all the RDF (or RDF and RDFS) axiomatic triples except those containing the container membership property IRIs <code>rdf:_1, rdf:_2, ...</code></li>
     <li>For every container membership property IRI which occurs in E, add the RDF (or RDF and RDFS) axiomatic triples which contain that IRI.</li>
+    <li>For every IRI <code>aaa</code> used in E, add <code>aaa rdf:type rdfs:Resource</code> to S.</li>
     <li>Apply the RDF (or RDF and RDFS) inference patterns as rules, adding each conclusion to the graph, to exhaustion; that is, until they generate no new triples.</li>
     <li>Determine if E has an instance which is a subset of the set, i.e. whether the enlarged set <a>simply entails</a> E.</li>
```
Comment (TallTed) [inline spec/index.html:1521]:
```suggestion
    <li>Determine if E has an instance which is a subset of the set, i.e., whether the enlarged set <a>simply entails</a> E.</li>
```

File: spec/index.html [hunk 2]
```diff
@@ -1607,6 +1608,7 @@ <h2>Entailment rules (Informative)</h2>
     <li>Add to S all the RDF (and RDFS) axiomatic triples which do not contain any container membership property IRI.</li>
     <li>For each container membership property IRI which occurs in E, add the RDF (and RDFS) axiomatic triples which contain that IRI.</li>
     <li>If no triples were added in step 2., add the RDF (and RDFS) axiomatic triples which contain <code>rdf:_1</code>.</li>
```
Comment (TallTed) [inline spec/index.html:1610]:
```suggestion
    <li>If no triples were added in step 2, add the RDF (and RDFS) axiomatic triples which contain <code>rdf:_1</code>.</li>
```

**Review by TallTed — CHANGES_REQUESTED** [review]
randomly noticed punctuation fixes

**Comment by pfps** [comment]
I tried to commit Ted's fixes, but now I can't see the entire PR.   Has something gone wrong?

**Review by TallTed — APPROVED** [review]

**Review by pfps — APPROVED** [review]
Looks OK

**Review by doerthe — APPROVED** [review]

**Pull request merged**